main.py

```diff
@@ -83,13 +84,16 @@ def extract(entity_json: str) -> Dict[str, Any]:
         response: File response for Fast API.
     """
     entities = json.loads(entity_json)
-    for name, description in entities.items():
-        entities[name] = Field(description=description)
+
     # Creating a new pydantic object from the entity
-    FormatModel = type("FormatModel", (BaseModel,), entities)
+    format_model = create_model("FormatModel", **entities)
 
-    extractor = ExtractionModel(format_model=FormatModel)
-    result = extractor.run(docs)[0]
-    return {
-        "items": result,
-    }
+    extractor = ExtractionModel(format_model=format_model)
+    results = extractor.run(docs)
+
+    # Generate a CSV file
+    csv_file = extractor.generate_csv(entities, results)
```

Comment:
this works. We could have also leveraged pandas here since entities is just a dict
```
import pandas as pd
df = pd.DataFrame(entities)
csv_file = StringIO()
df.to_csv(csv_file, index=False)
```

Reply:
Oh nice! I might do that instead, depending on if pandas is a reasonably sized dependency to add